lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -3910,18 +3910,36 @@ void request_disassemble(const llvm::json::Object &request) {
     return;
   }
 
-  addr_ptr += GetSigned(arguments, "instructionOffset", 0);
-  lldb::SBAddress addr(addr_ptr, g_dap.target);
+  addr_ptr += GetSigned(arguments, "offset", 0);
```

Comment:
Is "offset" required? If so we don't want to set it to zero if this fails, we would wan't to return an error. Use the above `GetSigned` function that returns an `llvm::Expected`